cli/util/fs.rs

```diff
@@ -261,6 +261,7 @@ pub struct FileCollector<TFilter: Fn(WalkEntry) -> bool> {
   ignore_git_folder: bool,
   ignore_node_modules: bool,
   ignore_vendor_folder: bool,
```

Comment:
This `ignore_vendor_folder` seems too broad and we should probably leave this up to users to ignore if they want to. Maybe something to do in 1.42?